test/onnx/gen_onnx.py

```diff
@@ -7417,22 +7438,132 @@ def resize_downsample_f_test():
 
 
 @onnx_test()
-def resize_downsample_c_test():
-    scales = np.array([1.0, 1.0, 0.6, 0.6], dtype=np.float32)
+def resize_downsample_f_dyn_test():
+    # scales is an input
+    # TODO:  How to create an Onnx file where scales is a runtime input?
```

Comment:
See the variable input slice onnx tests: https://github.com/ROCm/AMDMIGraphX/blob/05642ec7807e08e27a0efec3ddb1b364cd2cddbb/test/onnx/gen_onnx.py#L8738

Reply:
ok, new test added